eta/core/video.py

```diff
@@ -1854,46 +1854,26 @@ def sample_select_frames(
     Returns:
         a list of the sampled frames if output_patt is None, and None otherwise
     '''
-    # Parse parameters
-    resize_images = size is not None
+    if fast:
```

Comment:
If we are going to have logic that handles failures, I believe it should be here, not inside the private methods.

It's confusing that, as written, `_sample_select_frames_fast()` is really "try to do it fast, but there are two different cases in which fast won't work and we will secretly call `_sample_select_frames_slow()` here